packages/protocol/lib/proxy-utils.ts

```diff
@@ -31,27 +31,6 @@ export async function verifyProxyStorageProof(web3: Web3, proxy: string, owner:
   return proof.storageHash === bufferToHex(trie.root)
 }
 
-export async function retryTx(fn: any, args: any[]) {
```

Comment:
I agree with this change
sometimes its nice to "sanitize" the diff by not making a location change which touches a lot of files and distracts from the intention of the PR
a trick I learned from https://mtlynch.io/code-review-love/